privaterelay/management/commands/update_phone_remaining_stats.py

```diff
@@ -3,58 +3,88 @@
 from django.conf import settings
 from django.core.management.base import BaseCommand
 
-from allauth.socialaccount.models import SocialAccount
+import logging
+
+from emails.models import Profile
+from privaterelay.management.utils import (
+    get_free_phone_social_accounts,
+    get_phone_subscriber_social_accounts,
+)
 
 if settings.PHONES_ENABLED:
     from phones.models import RelayNumber
 
-from privaterelay.views import update_fxa
+logger = logging.getLogger("events")
+
+
+def reset_phone_remaining_stats(user):
+    # re-set remaining_texts and remaining_seconds to the maximum value
+    try:
+        relay_number = RelayNumber.objects.get(user=user)
+    except RelayNumber.DoesNotExist:
+        # no RelayNumber set, do nothing
+        return
+    relay_number.remaining_texts = settings.MAX_TEXTS_PER_BILLING_CYCLE
+    relay_number.remaining_seconds = settings.MAX_MINUTES_PER_BILLING_CYCLE * 60
+    relay_number.save()
 
 
-def update_phone_remaining_stats():
-    if not settings.PHONES_ENABLED:
-        return []
-    accounts_with_phones = []
-    for sub_with_phone in settings.SUBSCRIPTIONS_WITH_PHONE:
-        social_accounts = SocialAccount.objects.filter(
-            extra_data__icontains=sub_with_phone
+def get_next_reset_date(profile: Profile) -> datetime:
+    # TODO: consider moving this as a property in Profile model
```

Comment:
suggestion (non-blocking): maybe a first step is to move this into `privaterelay.management.commands.utils`? 

Or instead of a property on `Profile` we could make a custom method on the queryset, so we can optimize the query too. Something like `Profile.objects.need_phone_stats_update()`?

Reply:
I thought about moving this to utils but there is no where in the current code that needs this function. The TODO was there so that the next engineer who needs to use this can make the best decision to use this outside of the command. That next person could me, but that's a future me problem 😆